src/components/ProfilePage/AddressBox/AddressBox.tsx

```diff
@@ -51,17 +54,24 @@ export const AddressBox = ({ address, isEVM }: AddressBoxProps) => {
         <Image
           alt="Effigy Wallet Icon"
           src={
-            address && isEVM
-              ? `https://effigy.im/a/${address}.png`
-              : `https://effigy.im/a/${'jumper.eth'}.png`
+            imageLink
+              ? imageLink
+              : address && isEVM
+                ? `https://effigy.im/a/${address}.png`
+                : `https://effigy.im/a/${'jumper.eth'}.png`
```

Comment:
Maybe we could do something like:

`const imageLinkToUse = imageLink || `https://effigy.im/a/${address && isEVM ? address : 'jumper.eth'}.png`;`